stacks/core/base/base.yml

```diff
@@ -4,15 +4,15 @@ services:
   # Mainflux services
   users:
     image: mainflux/users:0.11.0
-    env_file: './env.d/mainflux-users.env'
+    env_file: '../dev/env.d/mainflux-users.env'
```

Comment:
Please, as we discussed before, remove this change from the commit.

Reply:
Done.